templates/taxi/manufacturer_form.html

```diff
@@ -0,0 +1,17 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
+
+{% block content %}
+    <h1>{{ object|yesno:"Update, Create" }} a Manufacturer</h1>
```

Comment:
You have an `h1` tag here. It's a good practice to have only one `h1` tag on a page for SEO purposes. Ensure that no other `h1` tags are present on the page where this template is used.